src/V3Simulate.h

```diff
@@ -962,30 +961,24 @@ class SimulateVisitor VL_NOT_FINAL : public VNVisitorConst {
     }
 
     void visit(AstJumpBlock* nodep) override {
-        if (jumpingOver(nodep)) return;
+        if (jumpingOver()) return;
         iterateChildrenConst(nodep);
+        if (m_jumpp && m_jumpp->blockp() == nodep) {
+            UINFO(5, "   JUMP DONE " << nodep);
+            m_jumpp = nullptr;
+        }
```

Comment:
Isn't a VL_RESTORED or something else needed here given jumpingOver now has no nodep?  How would this otherwise work?   Probably don't have a test like this currently.

```
JUMPBLOCK a
   ...
   JUMPBLOCK b
   ...
      if (x) JUMPGO b
      if (x) JUMPGO a
   ... 
   end b
   ...  // jumpgo a skips these, but jumpgo b does not
end a
```

Reply:
Incidentally I'm working on a proper CFG builder that you might just be able to fix this with...